packages/fiori/src/themes/DynamicPageTitle.css

```diff
@@ -128,7 +128,8 @@
 }
 
 /* focus */
-:host([focused]) {
+:host([desktop]:focus-within),
+:host(:focus-visible) {
     outline: var(--_ui5_dynamic_page_title_focus_outline);
     outline-offset: -0.0625rem;
```

Comment:
Could you please check how much exactly should the focus `outline-offset` be?
Looking at the [alternative example](https://openui5nightly.hana.ondemand.com/resources/sap/ui/documentation/sdk/index.html?sap-ui-xx-sample-id=sap.f.sample.DynamicPageFreeStyle&sap-ui-xx-sample-lib=sap.f&sap-ui-xx-sample-origin=.&sap-ui-xx-dk-origin=https%3A%2F%2Fopenui5nightly.hana.ondemand.com&sap-ui-theme=sap_horizon&sap-ui-rtl=false&sap-ui-density=sapUiSizeCompact) seems that we are using `2px` (0.125rem) as offset.